src/components/Info.js

```diff
@@ -0,0 +1,33 @@
+import Header from './Header';
+import { Link } from 'react-router-dom';
+
+const Info = ({ contacts }) => {
```

Comment:
You'll want to pass just one contact here (the selected one) or pass the array but filter for the selected contact.